clients/admin-ui/src/features/privacy-requests/DenyModal.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    Textarea,
+    Button,
+} from "@fidesui/react";
+import React from 'react';
+
+
+
+type DenyModalProps ={
+    isOpen: boolean
+    isLoading: boolean
+    handleMenuClose:  () => void
+    handleDenyRequest:  (reason: string) => Promise<any>
+    denialReason: string
+    onChange: (e:any)=> void
+
+}
+
+const DenyModal = ({ isOpen, isLoading,handleMenuClose, denialReason, onChange, handleDenyRequest}: DenyModalProps ) => (
```

Comment:
Actually, maybe this is a question for another discussion, but are we using "subject request" and "privacy request" interchangeably -- that seems like it could be confusing, especially in the code, very quickly? Maybe a question for @elliotbonneville or @seanpreston ?

Reply:
The component was renamed to `DenyPrivacyRequestModal`